pkg/maps/configmap/config_map.go

```diff
@@ -32,14 +32,21 @@ const (
 	AgentLiveness
```

Comment:
@dylandreimerink @jrajahalme it looks like the "marker" entry `UsedEntries` is unused and lost its meaning. When L2 Announcement has been introduced, a new entry has been added to the end (whereas the intention of the marker entry was to be the last entry).

should we delete the entry and move `AgentLiveness` to second entry?

Reply:
Moving `AgentLiveness` is alright with me, as long as we do it before the feature freeze. Moving after that could pose upgrade issues if the map is not recreated. I also have no idea what `UsedEntries` was supposed to be used for.